### ISSUE TITLE
[FE] Demographics Page - Button Styling

### ISSUE THREAD

**Issue by julieadair1** [department-of-veterans-affairs/va.gov-team#34668]
## How to configure this issue

- [ ] Attached to an Epic (what body of work is this a part of?)
- [ ] Correctly Labeled
- [ ] Fill in User Stories
- [ ] Fill in Documentation
- [ ] Fill in Tasks
- [ ] Fill in Acceptance Criteria

---

## User Story

As veteran I need to have the big button vs. the standard button when I click yes or no on the Demographics page so that I can easily click it (with my thumb for example). 

- (%FILL IN%)
- (%FILL IN%)

## Related Documentation

> these should be links to feature docs

- [Sketch](https://www.sketch.com/s/e79a827e-42cf-4a82-b554-874c75b5c70e)
- [Product Docs](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/checkin)
- [Feature Doc](%FILL IN%)

## Description

> this should be an understandable description of the ticket. This description should have enough context meaning that non-technical team members should get the idea.

## Tasks

> this is a list of technical tasks that need for this ticket

- [ ] %Add any tasks%
- [ ] Create or update unit tests
- [ ] Create or update e2e tests
- [ ] Update relevant documentations
  - [ ] README
  - [ ] Use Cases

## Acceptance Criteria

I am able to click the "yes" and "no" buttons using the big button styling instead of the standard button styling. 

- [ ] %Add any additional Acceptance Criteria%
- [ ] Passes Accessibility checks using axe
- [ ] Deployed to staging.
- [ ] Designer has signed off on the feature
- [ ] Product team has signed off on the  feature

**Comment by mdewey** [comment]
UX review: https://www.sketch.com/s/5a0f7c7f-df7c-4506-b4f9-c003321595fe/a/Gm5VG7w

**Comment by mdewey** [comment]
This is in staging and needs to be reviewed by the UX team 

CC: @niradatta1 @kristenmcconnell @ytsaoca @zach-park

**Comment by julieadair1** [comment]
@kristenmcconnell @ytsaoca @zach-park @niradatta1  were any of you able to validate this in staging? 
cc: @jparker0425

**Comment by zach-park** [comment]
> @kristenmcconnell @ytsaoca @zach-park @niradatta1 were any of you able to validate this in staging? cc: @jparker0425

I believe this item (#34668) was checked during the FE review meeting with @niradatta1 @ytsaoca. The ticket probably needs to be updated but needs confirmation by Nira or Ya-ching. :)

**Comment by niradatta1** [comment]
@kristenmcconnell yes, this was checked and approved during our FE review meeting.
